### PR TITLE
Add script for running tests in CI

### PR DESCRIPTION
**Pull request by lmars**
@SimonWoolf please modify to start the X server.

### CODE REVIEW

**Comment by SimonWoolf** [comment]
@lmars added

**Comment by lmars** [comment]
@SimonWoolf the build seems to be failing (see [here](https://ci.ably.io/job/ably-js/497/console)), thoughts? Is anything missing in terms of dependencies?

File: ci/run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+set -eo pipefail
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+
+main() {
+	echo "** Starting xvfb"
+	export DISPLAY=:99
+	/usr/bin/Xvfb $DISPLAY &
+	echo "** Started xvfb"
```
Comment (lmars) [inline ci/run-tests.sh]: 2 space indent please :grin:

Comment (SimonWoolf) [inline ci/run-tests.sh]: As some point I should lock you and Paddy in a room and force you to fight it out, winner gets to decide the tabs vs spaces policy :P

Comment (lmars) [inline ci/run-tests.sh]: I just blindly follow the [Google shell style guide](https://google.github.io/styleguide/shell.xml) for all scripts (so specifically [this](https://google.github.io/styleguide/shell.xml?showone=Indentation#Indentation)).

Comment (lmars) [inline ci/run-tests.sh]: P.S. all my Go programs use tabs :wink:

**Comment by SimonWoolf** [comment]
@lmars fixed by added ritual sacrifice to the npm gods, see https://ci.ably.io/job/ably-js/502/console

**Pull request merged**